src/services/lease-service/get-tenant.ts

```diff
@@ -18,11 +18,17 @@ type GetTenantError =
   | 'housing-contracts-not-found'
   | 'get-lease-property-info'
 
+// TODO: Tenant should somehow represent that is definitely has at least one
+// active housing contract. Otherwise, it wouldn't be a tenant, right?
+
+type NonEmptyArray<T> = [T, ...T[]]
+
 export type Tenant = Omit<Contact, 'leases' | 'isTenant'> & {
```

Comment:
Can we export this type to onecore-types?

Reply:
Will apply after this is merged:
https://github.com/Bostads-AB-Mimer/onecore-types/pull/87

Reply:
https://github.com/Bostads-AB-Mimer/onecore-leasing/pull/109/commits/9165a8b7b33eaed323470903446f2da1c87921ef